package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "budget-app-node",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node src/index.js",
+    "dev": "nodemon src/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/konradperko/BudgetApp-Node.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/konradperko/BudgetApp-Node/issues"
+  },
+  "homepage": "https://github.com/konradperko/BudgetApp-Node#readme",
+  "dependencies": {
+    "express": "^4.17.1"
```

Comment:
Czy tutaj powinno być tak że sciąga najnowszą? Bo nie ktore rzeczy mogą wygladać w nowszych wersjach.

Reply:
narazie IMHO powinno, jak bedziemy mieli jakas produkcyjna wersje dla wielu uzytkownikow to mozna pomyslec zeby to zamrozic